stub.py

```diff
@@ -84,28 +84,21 @@ def get_locations(locations, new_page):
 
         return locations
 
+    def get_seattraits(self, event_id, params={}):
+        '''get seattraits for an event'''
 
-            # def search_seattraits(self, params={}):
-            #     '''Search for events in a particular seattraits'''
-            #     allowed_params = {
-            #         'id',
-            #         'name',
-            #         'seatTraits',
-            #         'seatTraits.id',
-            #         'seatTraits.name',
-            #         'seatTraits.type',
-            #     }
-            #     url = self.set_url('/partners/catalog/events/v3/{eventId}/seatTraits')
-            #     params.setdefault('rows', '500')
-            #
-            #     def get_seattraits(seattraits, new_page):
-            #         seattraits['seattraits'].extend(new_page['seattraits'])
-            #         return seattraits
-            #
-            #     seattraits = self._get_pages(url, params, allowed_params, get_seattraits,
-            #                                 'Could not retrieve seattraits.')
-            #
-            #     return seattraits
+        url = self.set_url('/partners/catalog/events/v3/'+ event_id + '/seatTraits')
+
+        def save_seattraits(seattraits, new_page):
+            seattraits['seattraits'].extend(new_page['seattraits'])
+            return seattraits
+
+        def _get_pages(url, params , page_func, error_msg):
+            return {}
```

Comment:
This is where you've gone wrong, you massive egg. This function just returns an empty dictionary every time! You need to use something like
```
seats = requests.get(url, headers=self.headers)
return seats.json()
```
I meant for you to use my `_get_pages` as a reference, not to re-write it. You also don't need the `save_seattraits` function.